### PR TITLE
Save Fields from Tracer Reader to Field Manager

### PR DESCRIPTION
**Pull request by odiazib**

### CODE REVIEW

File: components/eamxx/tests/single-process/mam/aero_microphys/output.yaml [hunk 1]
```diff
@@ -6,7 +6,10 @@ Fields:
   Physics:
     Field Names:
       - T_mid
-      
+      - oxi_fields
+      - linoz_fields
+      - vertical_emission_fields
+
```
Comment (singhbalwinder) [inline components/eamxx/tests/single-process/mam/aero_microphys/output.yaml:12]: This flag will eventually be "off" by default. Will these added fields cause the model to crash if this flag is not defined? If yes, we can remove them after initial evaluation.

**Review by singhbalwinder — CHANGES_REQUESTED** [review]
Looks good! I requested a minor change.

File: components/eamxx/src/physics/mam/eamxx_mam_microphysics_process_interface.cpp [hunk 2]
```diff
@@ -22,6 +22,8 @@
 #include <ekat/kokkos/ekat_subview_utils.hpp>
 #include <iostream>
 
+#define SCREAM_MAM4xx_OUTPUT_TRACER_FIELDS
```
[the remark this answers is not in the document]
Comment (odiazib) [reply components/eamxx/src/physics/mam/eamxx_mam_microphysics_process_interface.cpp:25]: I called `ENABLE_OUTPUT_TRACER_FIELDS`

File: components/eamxx/tests/single-process/mam/aero_microphys/output.yaml (hunk 1 above)
Comment (odiazib) [reply components/eamxx/tests/single-process/mam/aero_microphys/output.yaml:12]: I commented out these fields and added a note.

**Review by singhbalwinder — APPROVED** [review]

**Pull request merged**